.github/CODEOWNERS

```diff
@@ -58,7 +58,6 @@
 # API report changes
 /**/api-report/*.public.api.md                 @microsoft/fluid-cr-api
 /**/api-report/*.beta.api.md                   @microsoft/fluid-cr-api
-/**/api-report/*.legacy.*.api.md               @microsoft/fluid-cr-api
```

Comment:
this just included legacy alpha, which is not needed now, as the public and beta rule above will appropriately cover legacy for public and beta, and legacy alpha no longer needs api review

Reply:
Yay! Less special-ness!